packages/web/tailwind.config.js

```diff
@@ -220,6 +220,7 @@ module.exports = {
         earnpositions: "minmax(0, 1fr), 1px, minmax(0, 284px)",
       },
       width: {
+        25: "6.25rem",
```

Comment:
In general, IMO we should avoid updating the global tailwind config to keep it lean and organized. I think it's OK to hardcode values if absolutely needed. Won't die on this hill though. Thoughts?

Reply:
Hardcode values can be fine but we should avoid them, having everything in configuration allows us to make changes without having to check all the files. I think we will generally solve this problem when we have a stable design system configuration (and consequently tailwind as well).